test/e2e-encryption-perf/encryption_perf_test.go

```diff
@@ -27,7 +27,7 @@ const (
 func TestPerfEncryptionTypeAESCBC(tt *testing.T) {
 	ctx := context.TODO()
 	clientSet := getPerfClients(tt)
-	library.TestPerfEncryptionTypeAESCBC(tt, library.PerfScenario{
+	library.TestPerfEncryption(tt, library.PerfScenario{
```

Comment:
you'll need to set the new EncryptionProvider field of PerfScenario  to `configv1.EncryptionTypeAESCBC`